src/server/request_context.h

```diff
@@ -72,6 +72,9 @@ class RequestContext {
         std::istringstream stream(get_argument(name));
         T v;
         stream >> v;
+        if(stream.fail()) {
+          throw std::runtime_error("Invalid argument");
+        }
         return v;
     }
```

Comment:
Better reuse `kiwix::extractFromString<T>()` and implement all error handling in the latter.